chronos2/dashboard_backend/.env.sample

```diff
@@ -1,12 +1,12 @@
-DB_USER
-DB_PASSWORD
-POSTGRES_PASSWORD
-DB_HOST
-DB_PORT
-DB_NAME
+POSTGRES_DB=postgres
```

Comment:
I've set defaults here to make it easier to get up and running with the docker deployment